sound/soc/sof/sof-client.h

```diff
@@ -0,0 +1,80 @@
+/* SPDX-License-Identifier: (GPL-2.0) */
+/*
+ * This file is provided under a dual BSD/GPLv2 license.  When using or
+ * redistributing this file, you may do so under either license.
```

Comment:
Is this correct? SPDX says "GPL-2" and this comment says BSD/GPLv2? In the next patch you hhave "// SPDX-License-Identifier: (GPL-2.0 OR BSD-3-Clause)"